dev-build/conf/routes

```diff
@@ -120,6 +120,9 @@ GET            /video/end-slate/series/*seriesId.json
 GET            /video/end-slate/section/*sectionId.json                                                                          controllers.VideoEndSlateController.renderSection(sectionId)
 GET            /embed/card/*path.json                                                                                            controllers.RichLinkController.render(path)
 GET            /embed/card/*path                                                                                                 controllers.RichLinkController.renderHtml(path)
+GET            /embed/contentcard/*path.json                                                                                     controllers.ContentCardController.render(path, group: Option[Int], card: Option[Int])
+GET            /embed/contentcard/*path                                                                                          controllers.ContentCardController.renderHtml(path)
```

Comment:
I pushed a change to make it so that neither will accept the query params since there is currently no need for them.